Components/image/image.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Component-Library</title>
+      <link rel="stylesheet" href="/styles.css">
+      <link rel="stylesheet" href="image.css">
+</head>
+<body>
+      <div class="sidenav">
+            <a href="/index.html">Introduction</a>
+            <a href="/avatar/avatar.html">Avatar</a>
+            <a href="/alert/alert.html">Alert</a>
+            <a href="/badge/badge.html">Badge</a>
+            <a href="/button/button.html">Button</a>
+            <a href="/card/card.html">Card</a>
+            <a href="#" class="focus">Image</a>
+            <a href="/input/input.html">Input</a>
+            <a href="/text-utilities/text-utilities.html">Text-Utilities</a>
+            <a href="/list/list.html">List</a>
+            <a href="/navigation/navigation.html">Navigation</a>
+      </div>   
+      <div class="main">
+            <h2>Image Component in Component-Library</h2> 
+            <div>
+                  <h3>Responsive Image</h3>
+          
+                  <div>
+                        <img class="image-responsive" src="2550833-bigthumbnail.jpg" alt="image3">
+                  </div>
+                  <h3>Round Image</h3>
+                  <div class="VariantContainer no-border">
+                      <img class="image-circle" src="2550833-bigthumbnail.jpg" alt="image2">
+                  </div>
```

Comment:
dont use jpg , png , you can use webp instead for 

- Better Performance of the website
- Compressed Size
- Good Maintained Quality of the images